guide/how-it-works/ecash/design-best-practices.md

```diff
@@ -0,0 +1,573 @@
+   alt-text = "Example of important announcements display in a wallet interface"
+   width = 250
+   height = 390
+   layout = "float-right-desktop -background -shadow"
+   caption = "Supported features are pulled by indexing a mints information and reading what nuts the mint has implemented."
+%}
+
+- `nuts`: Indicates which NUT specifications the mint supports and their settings. Create a "Features" or "Capabilities" section in the mint details, translating technical NUT numbers into user-friendly feature descriptions.
+
+</div>
+
+{% include tip/open.html color="green" icon="check" label="Avoid Jargon & NUTs 01-06" %}
+
+When displaying [NUTs](https://github.com/cashubtc/nuts) (Notation, Usage, and Terminology) that a mint supports, avoid simply listing NUT01, NUT02, etc. Instead, provide users with clear descriptions of each NUTs functionality. This helps users understand the features and capabilities of the mint at a glance. Since NUTs 01-06 are mandatory for Cashu protocol interoperability, focus on optional NUTs that add unique functionality. Consider using the descriptions provided in the official documentation to keep the information accurate.
+
+{% include tip/close.html %}
+
+#### Organizing Cashu Metadata Fields
+When designing your Cashu wallet, consider organizing these metadata fields in a logical, user-friendly manner to help users understand and interact with different mints. Here are some suggestions:
+
+- Use the `name` and short `description` in mint listings for quick identification of mints.
+- Display a detailed mint information page using `description_long`, `version`, and `contact` details.
+- Display the `motd` as a prominent notification when relevant to the specific mint.
+- Use the `nuts` information to create a features list that helps users understand each mint's capabilities.
+
+By effectively incorporating these Cashu metadata fields, you can create a more transparent and informative experience for users, helping them make informed decisions about which Cashu mints to trust and use. This is particularly important for Cashu, as users interact directly with individual mints rather than a federation structure like in Fedimint.
+
+### Advance Features
+
+#### Pay to Public Key (P2PK)
+
+{% include image-gallery.html pages = page.images_p2pk %}
+
+[NUT11](https://github.com/cashubtc/nuts/blob/main/11.md) is a powerful feature that allows bitcoin-backed ecash tokens to be securely locked to another user's public key, which is generated by the recipient's wallet. This ensures that only the intended recipient can redeem the ecash. NUT11 enables secure offline payments, preventing double-spending. Beyond these basics, NUT11 supports advanced use cases like timelocks and multisignature (multisig) setups, where ecash can be conditionally spent or jointly owned by multiple parties. When designing make sure these functionalities are clearly communicated to users, highlighting their practical benefits and flexibility.
```

Comment:
I’m not sure I understand. Could you explain it again? Is this potentially a new feature or use case? If so, it might fall outside the scope of the design guide if it doesn’t already exist. As far as I know, no one has included any GPG key functionality in ecash apps.

Reply:
A send-to-contact feature based on Nostr or GPG keys might work as follows: user creates a contact list storing the various user's public keys (or just pick them up from nostr). When sending funds the app UI could offer a send to contact feature what would lock the token to the intended recipient (contact)'s public key so only that they can spend it. I think thats what NUT-11 is for...dunno but I hope this is clearer or just ping me and we can chat.

Reply:
Sounds good, I'll message you.